### PR TITLE
St maps

### PR DESCRIPTION
**Pull request by andy6a**
Add new and improved visuals for a variety of situational uses when assessing sourcetracker results. Adjustments to readme for current archive location.

### CODE REVIEW

File: sourcetracker/_cli/gibbs.py [hunk 1]
```diff
@@ -185,6 +208,24 @@ def gibbs(table_fp: Table,
     fig, ax = plot_heatmap(mpm.T)
     fig.savefig(os.path.join(output_dir, 'mixing_proportions.pdf'), dpi=300)
```
Comment (cherman2) [inline sourcetracker/_cli/gibbs.py:210]: Is this being left in? Is this different from your heatmap?

File: sourcetracker/_plot.py [hunk 2]
```diff
@@ -23,3 +24,261 @@ def plot_heatmap(mpm, cm=plt.cm.viridis, xlabel='Sources', ylabel='Sinks',
     ax.set_ylabel(ylabel)
     ax.set_title(title)
     return fig, ax
```
Comment (cherman2) [inline sourcetracker/_plot.py:26]: again I am wondering if this is worth having in there twice

File: sourcetracker/_plot.py [hunk 3]
```diff
@@ -23,3 +24,261 @@ def plot_heatmap(mpm, cm=plt.cm.viridis, xlabel='Sources', ylabel='Sinks',
...
+        ax.set_xlabel(xlabel)
+        ax.set_ylabel(ylabel)
+        ax.set_title(self.title)
+        plt.xticks(rotation=45, ha='right')
+        if unknowns:
+            plt.savefig(os.path.join(self.file,
+                                     self.title + "_heatmap.png"))
+        else:
+            plt.savefig(os.path.join(self.file,
+                                     self.title + "_heatmap_nounknown.png"))
```
Comment (cherman2) [inline sourcetracker/_plot.py:60]: When we meet can you explain the reason that you are checking for whether there are unknowns? I dont fully understand

File: sourcetracker/_plot.py [hunk 4]
```diff
@@ -23,3 +24,261 @@ def plot_heatmap(mpm, cm=plt.cm.viridis, xlabel='Sources', ylabel='Sinks',
...
+        normalized=True means that you wish to normalize
+        each column to equal to 1 to represent the likelihood
+        that each member of the pair can be mapped its'
+        counter out of 1.
+        instead of less than 1. I recommend to do this
+        to get a better idea of the proportion. Also recommend if getting
+        rid of
+        unknowns or transposing the data to see if these improve your plot.
+
+        Unknown=False means to normalize this plot without the unknown column.
```
Comment (cherman2) [inline sourcetracker/_plot.py:74]: Oh this is the math you are doing with the unknowns. Ignore my above comment

File: sourcetracker/_plot.py [hunk 5]
```diff
@@ -23,3 +24,261 @@ def plot_heatmap(mpm, cm=plt.cm.viridis, xlabel='Sources', ylabel='Sinks',
...
+        high unknowns from letting you misidentify a successful match
+
+        I would recommend running both separately
+        and together in order to eliminate marginal cases.
+
+        Any analysis should be done using a bin(n,x)
+        distributions to better give a good idea of these distributions.
+
+        Color=viridis spans from dark purple to yellow through blue and green.
+
```
Comment (cherman2) [inline sourcetracker/_plot.py:85]: I like this explanation of colors.

File: sourcetracker/_cli/gibbs.py [hunk 6]
```diff
@@ -123,6 +124,21 @@
...
+@click.option('--paired_heatmap', required=False, default=False, is_flag=True,
+              show_default=True)
+@click.option('--title', required=False, default='Mixing Proportions',
+              type=click.STRING, show_default=True)
+@click.option('--color', required=False, default='viridis', type=click.STRING,
+              show_default=True)
+@click.option('--unknowns', required=False, default=True, is_flag=True,
+              show_default=True)
+@click.option('--transpose', required=False, default=False, is_flag=True,
+              show_default=True)
```
[the remark this answers is not in the document]
Comment (cherman2) [reply sourcetracker/_cli/gibbs.py:141]: For posterity, we discussed off line and you can pass them all.

File: sourcetracker/_cli/gibbs.py (hunk 1 above)
Comment (cherman2) [reply sourcetracker/_cli/gibbs.py:210]: Again, Discussed off line but this will be removed before merging.

**Comment by cherman2** [comment]
Looks like there are some import issues. 
Screenshot 2024-01-31 at 3 26 35 PM

**Comment by andy6a** [comment]
Hey Chloe, I have 2 options here, first is I can just pull those lines out for now as there is no plot_heatmap anymore (although you were already having issues getting plots made so I am unsure as to the functionality here.) The second option is to try to modify to  my new heatmap but I don't know how that would work. Let me know!

**Comment by cherman2** [comment]
I think option one of removing those imports makes the most sense to me.

**Comment by andy6a** [comment]
Hey Chloe added the new function. Let me know what you think

**Comment by andy6a** [comment]
Closing in favor of splitting pull request into upgraded plots and new plots function seperately

**Pull request closed without merging**